docs/workforce-management-framework/topics/pro-keycloak-enablement.adoc

```diff
@@ -57,12 +57,10 @@ See xref:{context}-keycloak-configuration-production[{Keycloak} Configuration fo
 [id='{context}-create-own-keycloak-realm']
 [discrete]
 === Creating your own {Keycloak} Realm
-IMPORTANT: Ensure that you have a deep understanding of {Keycloak}, Node.js and Javascript Adapters before attempting to change any of the configurations below.
```

Comment:
This seemed like a scary and not very useful statement

Reply:
This may be rephrased. We just wanted to warn people to not change things there unless they know what they doing.

Reply:
:+1: I'll try to rephrase instead of removing